zerver/tests/test_management_commands.py

```diff
@@ -112,4 +112,4 @@ def test_realm_creation_with_expired_link(self):
 
             result = self.client.get(generated_link)
             self.assertEquals(result.status_code, 200)
-            self.assertIn("The organization creation link has been expired or is not valid.", result.content)
+            self.assertIn("The organization creation link has been expired or is not valid.", result.content.decode('utf-8'))
```

Comment:
Ahh, that makes sense.  I would just have this inherit from AuthedTestCase.  I think we should eventually rename that to `ZulipTestCase`, since I think that better described what it actually does...